Makefile

```diff
@@ -56,6 +56,7 @@ install:
 	npm --prefix es-bs-base install
 	npm --prefix es-ds-components install
 	npm --prefix es-ds-docs install
+	(cd es-ds-components && npx nuxt build)
```

Comment:
I'm not sure this is the right approach for the TypeScript configuration, because it will remain unchanged while the dev is working on it. My PR for 1724 [has a different approach](https://github.com/EnergySage/es-ds/pull/1467/files#diff-643836dcb5b4e80004b3f0d1aa3cfe33907998cd299d9a66266f8c770917dc1e) where the `tsconfig.json` for `es-ds-components` is set as:
```
{
  "extends": "../es-ds-docs/.nuxt/tsconfig.json"
}
```
This aligns with how Nuxt's example repo sets it up, and also improves performance of the install command compared to this approach. Essentially, the typescript configuration in es-ds-components (which AFAIK is only used by IDEs anyways) is delegated to the docs app.